src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -672,6 +689,11 @@ message JobCapacityUpdate {
     Capacity Capacity = 2;
 }
 
+message JobCapacityValues {
```

Comment:
It's a single value, right? Seems like the message name doesn't need to be plural.